src/pages/InvestScreen.jsx

```diff
@@ -159,6 +172,27 @@ const InvestScreen = ({
                             <Line/>
                         </>
                     )}
+
+                    {/*    Rebalance */}
+                    {!rebalancedStocks && <Button
+                        style={common.button}
+                        loading={getTopYieldLoading}
+                        onPress={() => {
+                            onPressRebalance()
+                        }}>
+                        <Text style={common.buttonText}>Rebalance</Text>
+                    </Button>}
+                    {rebalancedStocks &&
+                    (
+                        <>
+                            <ListStock titles={["Symbol", "Price", "Yield", "Quantity"]}
+                                       loading={getTopYieldLoading}
+                                       stocks={suggestedStocks}
+                                       renderItem={renderTopStocks}
+                            />
+                            <Line/>
+                        </>
+                    )}
```

Comment:
I saw that you create rebalancedStocks = null on line 65. But you didn't update any value for it. Why you have the condition using rebalancedStocks in here?